packages/devtools_testing/lib/support/flutter/timeline_test_data.dart

```diff
@@ -1001,3 +1001,22 @@ final durationEventsWithDuplicateTraces = [
   transformLayerEnd1,
   endGpuRasterizerDrawTrace,
 ];
+
+final testTimelineJson = {
+  'type': 'Timeline',
+  'traceEvents': [
+    {
+      'name': 'thread_name',
+      'tid': testUiThreadId,
+      'args': {'name': '1.ui'},
+    },
+    {
+      'name': 'thread_name',
+      'tid': testRasterThreadId,
+      'args': {'name': '1.raster'},
+    },
+    ...goldenTraceEventsJson,
+  ],
+  'timeOriginMicros': 118039650802,
+  'timeExtentMicros': 118039679873 - 118039650802,
```

Comment:
I like how you wrote timeExtentMicros as the delta between the two other hard coded times!